docs/source/user-guide/latest/expressions.md

```diff
@@ -201,6 +201,19 @@ incompatible expressions.
 | VariancePop   |            | Yes                       |                                                                  |
 | VarianceSamp  |            | Yes                       |                                                                  |
 
+## Window Functions
+
```

Comment:
@andygrove I'm preparing set of tests today to get real status for windows, let me get back to this PR later today